README.md

```diff
@@ -18,7 +18,36 @@ This Digital Twin Definition Language (DTDL) Model Generator parses your DTDL js
 - *Namespace*: The namespace that will be injected into the generated model classes.
 - *JsonModelsDirectory*: The directory that contains the DTDL json files that we'll be parsing.
 - *OutputDirectory*: The directory that the generated model classes will be placed in.
-- *IncludeTemplateProject*: Whether or not to include the template project in the output directory.
+- *CopyrightHeader*: The header comment that contains copyright information.
+
+## Prerequisites
+The following are some Prerequisites/assumptions to be considered:
+
+For Developers
+- .Net 6 must be installed, while building the model generator locally. Backward compatibility is not supported.
+- The system should have GIT installed as the model generator uses MinVer package which requires GIT.
```

Comment:
Nit: Git is not an acronym, so it shouldn't be all capital letters.

Reply:
ReadMe Updated.